content/news/this-week-in-redox-15.md

```diff
@@ -0,0 +1,173 @@
+- [@bertinatto](https://github.com/bertinatto) added the `unique` command. Details [here](https://github.com/redox-os/coreutils/pull/143)
+- [@simondesloges](https://github.com/simondesloges) added the `-f` and `-F` options to `tail`. Details [here](https://github.com/redox-os/coreutils/pull/144)
+- [@simondesloges](https://github.com/simondesloges) changed `ls` to use `BufWriter`. Details [here](https://github.com/redox-os/coreutils/pull/147)
+- [@vladimiroff](https://github.com/vladimiroff) made `wc` consider Unicode spaces when counting words. Details [here](https://github.com/redox-os/coreutils/pull/150)
+- [@jackpot51](https://github.com/jackpot51) reimplemented `shutdown` on top of the `kill` system call.
+
+# TFS
+
+[TFS](github.com/redox-os/tfs) is a modular, fast, and feature rich next-gen file system, employing modern techniques for high performance, high space efficiency, and high scalability.
+
+There has been enough work done in TFS that we added this section in this edition so you can have an idea about how the progress is going there.
+
+First of all: Great news! [@ticki](https://github.com/ticki) has been working on many aspects of TFS and he considers that it is pretty near to functional. One of the great things is that as a product of this work several crates with interesting data structures and algorithms were created:
+
+- [chashmap](https://github.com/redox-os/tfs/tree/master/chashmap) - Concurrent hashmaps.
+- [cbloom](https://github.com/redox-os/tfs/tree/master/cbloom) - Concurrent bloom filters.
+[lz4](https://github.com/redox-os/tfs/tree/master/lz4) - An implementation of lz4
+- [speck](https://github.com/redox-os/tfs/tree/master/speck) - An implementation of SPECK cipher.
+- [tm](https://github.com/redox-os/tfs/tree/master/tm) - transactional memory.
+- [atomic-hashmap](https://github.com/redox-os/tfs/tree/master/atomic-hashmap) - Atomic hashmaps (you can read the [blog post](http://ticki.github.io/blog/an-atomic-hash-table/)).
+- [mlcr](https://github.com/redox-os/tfs/tree/master/mlcr) - A machine-learning based cache replacement strategy.
+- [seahash](https://github.com/redox-os/tfs/tree/master/seahash) - A hash function.
+- [concurrent](https://github.com/redox-os/tfs/tree/master/concurrent) - Hazard pointers implementation.
+- TFS has a logo.
+- There is a specification document.
+- A bunch of blog posts on design: https://github.com/redox-os/tfs#resources-on-design.
+
+And according [@ticki](https://github.com/ticki) there's a ton of interesting things to write about, and eventually he will be doing an in-depth blog post about them.
+
+## Package Management
+
+There has been a ton of work on this topic specially on the (pkgutils)[https://github.com/redox-os/pkgutils]. RedoxOS packaging utilities and in the [cookbook](https://github.com/redox-os/cookbook) the collection of package recipes.
```

Comment:
"RedoxOS" -> "Redox OS". As far as I know that's the preferred short form of the name.